app/brave_main_delegate.cc

```diff
@@ -145,6 +146,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       autofill::features::kAutofillServerCommunication.name,
       features::kAudioServiceOutOfProcess.name,
       features::kDefaultEnableOopRasterization.name,
+      network::features::kNetworkService.name,
```

Comment:
I posted an issue here to revert this and support the network service:
https://github.com/brave/brave-browser/issues/3819